build.sbt

```diff
@@ -312,15 +335,36 @@ lazy val stagedRelease = (project in file("core/src/test"))
 
 import ReleaseTransformations._
 
-// Don't use sbt-release's cross facility	
+// Don't use sbt-release's cross facility
 releaseCrossBuild := false
 
-def crossReleaseStep(step: ReleaseStep): Seq[ReleaseStep] = {
+lazy val downVersionPySpark = taskKey[Unit]("Replace PySpark")
+downVersionPySpark := {
+  "./downversion-pyspark.sh" !
```

Comment:
You've checked in a `txt` I believe - can you double-check?

Reply:
I removed the sh file and inlined the script.